Tests/SentryTests/Integrations/SentrySwiftIntegrationInstallerTests.swift

```diff
@@ -0,0 +1,61 @@
+@_spi(Private) @testable import Sentry
+@_spi(Private) import SentryTestUtils
+import XCTest
+
+final class SentrySwiftIntegrationInstallerTests: XCTestCase {
+
+    private var testHub: TestHub!
+    private var options: Options!
+    private var logOutput: TestLogOutput!
+
+    override func setUp() {
+        super.setUp()
+
+        // Create test hub
+        testHub = TestHub(client: nil, andScope: nil)
+        SentrySDKInternal.setCurrentHub(testHub)
+
+        // Create options
+        options = Options()
+        options.dsn = TestConstants.dsnAsString(username: "SentrySwiftIntegrationInstallerTests")
+        options.debug = true
+
+        // Setup log output
+        logOutput = TestLogOutput()
+        SentrySDKLog.setLogOutput(logOutput)
+        SentrySDKLog.configureLog(true, diagnosticLevel: .debug)
+    }
+
+    override func tearDown() {
+        clearTestState()
```

Comment:
`m`: I would really try to avoid using this method if we can. It does so many things. I would rather unset only what is required.

Reply:
Updated